dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/.classpath

```diff
@@ -1,9 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/TxFaultTolerance.war/src"/>
 	<classpathentry kind="src" path="test-applications/CDIFaultTolerance.war/src"/>
 	<classpathentry kind="src" path="test-applications/faulttolerance.jar/src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="/com.ibm.websphere.javaee.cdi.2.0/generated/com.ibm.websphere.javaee.cdi.2.0.jar">
```

Comment:
This isn't right ... should pull the api stuff in via the bnd container. Only the test app src folder needs adding here.